src/enclave/Enclave/ExpressionEvaluation.h

```diff
@@ -205,6 +206,18 @@ flatbuffers::Offset<tuix::Field> eval_binary_comparison(
         + std::string(" on ")
         + std::string(tuix::EnumNameFieldUnion(left->value_type())));
     }
+  } else {
+    // This code block handles comparison when at least one value is NULL.
+    // The logic can be summarized as: (note that the != operation implements XOR for boolean values)
+    // If is_null_asc = 1
+    // (x, NULL) = (0, 1) => (1, 0), (NULL, x) => (0, 1), (NULL, NULL) => (0, 0)
```

Comment:
I'm not following the meaning of this line. Perhaps it would be better expressed as a table? Alternatively we could remove it.

Reply:
Changed to a table.